docs/detections/api/rules/rules-api-bulk-actions.asciidoc

```diff
@@ -265,6 +276,14 @@ to apply.
 Yes, if action is `edit`.
 |==============================================
 
+[[bulk-actions-rules-api-dry-run]]
+[discrete]
+==== Dry run mode
+Enable dry run mode to allow test runs when applying bulk actions to rules. Dry runs can help you verify that the rules you specified can actually be updated. They're also useful for testing bulk rule updates before they're permanently applied. When dry mode is enabled, bulk actions are applied to specified rules to show you a realistic set of results. Rule updates are not written to {es}.
```

Comment:
```suggestion
Enable dry run mode to verify that bulk actions can be applied to specified rules. Certain rules, such as prebuilt Elastic rules, can't be edited and will return errors in the request response. Error details will contain an explanation, the rule name and/or ID, and additional troubleshooting information. 

To enable dry run mode on a request, add the query parameter `dry_run=true` to the end of the request URL. Rules specified in the request will be temporarily updated. These updates won't be written to {es}.
```

Reply:
@vitaliidm @banderror hopefully this revision is more accurate. Feel free to tweak it as needed!

Also, if you end up using this, you can remove the tip in line 284. I included it in the revision above.

Reply:
That version looks perfect 🙏